### PR TITLE
Fixes #9 support opensource.org and spdx.org license urls

### LINKED ISSUE #9: Error: "unknown license"
It is throwing, it should simply specify that it not a valid license.

ouput:

````
C:\proj> licensecheck
C:\Users\giggi\AppData\Roaming\nvm\v5.2.0\node_modules\licensecheck\index.js:170
        throw Error("unknown license: " + JSON.stringify(license))
        ^

Error: unknown license: {"url":"http://opensource.org/licenses/mit-license.php"}
    at Error (native)
    at formatLicense (C:\Users\giggi\AppData\Roaming\nvm\v5.2.0\node_modules\licensecheck\index.js:170:15)
    at Array.map (native)
    at checkPath (C:\Users\giggi\AppData\Roaming\nvm\v5.2.0\node_modules\licensecheck\index.js:209:48)
    at C:\Users\giggi\AppData\Roaming\nvm\v5.2.0\node_modules\licensecheck\index.js:247:19
    at Array.forEach (native)
    at checkPath (C:\Users\giggi\AppData\Roaming\nvm\v5.2.0\node_modules\licensecheck\index.js:246:56)
    at Object.<anonymous> (C:\Users\giggi\AppData\Roaming\nvm\v5.2.0\node_modules\licensecheck\cli.js:52:40)
    at Module._compile (module.js:399:26)
    at Object.Module._extensions..js (module.js:406:10)
````

### PR DESCRIPTION
**Pull request by alex-jose**
Added support for opensource.org and spdx.org license urls for fixing #9 .
Unknown licenses are specified as "unknown" instead of throwing error.

### CODE REVIEW

File: urltolicense.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+// Check if url is an spdx.org license url
+function isspdxLicense(url) {
+  re = /(http(s)?\:\/\/)?(www.)?spdx.org\/licenses\/[A-Za-z0-9]|[\-\.\_]\.html$/
+  return url.match(re)
+}
+
+// Find and return license name from spdx.org url
+function getspdxLicense(url) {
```
Comment (marcello3d) [inline urltolicense.js]: can you do this using the same regex you have above?

File: urltolicense.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+// Find and return license name from opensource.org url
+function getopensourceorgLicense(url) {
+  var licensenames = {
+    "mit-license.php": "MIT",
+    "gpl-license"    : "GPL",
+    "lgpl-license"   : "LGPL"
+  }
+  var array = url.split("/");
+  var license = array[array.length - 1];
```
Comment (marcello3d) [inline urltolicense.js]: can this be pulled out from the regex you evaluated already?

**Comment by marcello3d** [comment]
This PR reminds me that I need to add standard.js eslint to this project. :-)

If it's convenient, can you try to match the codestyle of the existing code? (It should be pretty close to standard.js style—I'll have to do a migration soon.)

**Comment by alex-jose** [comment]
Sorry for my bad coding style. 
I have simplified and modified my code to match with existing code style. Please review my changes.

**Comment by marcello3d** [comment]
Nice, thanks!

**Pull request merged**